src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Commands/SubmitProviderResponse/SubmitProviderResponseCommandHandler.cs

```diff
@@ -51,10 +48,19 @@ public async Task<SubmitProviderResponseResponse> Handle(SubmitProviderResponseC
             var templateId = _options.Value.NotificationTemplates.FirstOrDefault(p => p.TemplateName == "RATProviderResponseConfirmation")?.TemplateId;
```

Comment:
This is not using the EmailTemplates - I have re-declared them on my branches

Reply:
Added EmailTemplateNames in